### ISSUE TITLE
Class not found error

### ISSUE THREAD

**Issue by cookieguru** [eclipsesource/tabris-maps#62]
### Problem description

Upon instantiating the map, the app crashes.  The map displays normally when the app is built in debug mode.

```plain
08-26 00:01:56.600 29511 29511 E e.fire.dispatc: No package ID 30 found for ID 0x30a3d730.
08-26 00:01:56.642 29511 29511 I zzbz    : Making Creator dynamically
08-26 00:01:56.644 29511 29511 W e.fire.dispatc: Unsupported class loader
08-26 00:01:56.645 29511 29511 W e.fire.dispatc: Skipping duplicate class check due to unsupported classloader
08-26 00:01:56.646 29511 29511 I DynamiteModule: Considering local module com.google.android.gms.maps_dynamite:0 and remote module com.google.android.gms.maps_dynamite:221
08-26 00:01:56.646 29511 29511 I DynamiteModule: Selected remote version of com.google.android.gms.maps_dynamite, version >= 221
08-26 00:01:56.655 29511 29511 W ChimeraDebugLogger: Singleton logger instance not set.
08-26 00:01:56.658 29511 29511 W e.fire.dispatc: Unsupported class loader
08-26 00:01:56.661 29511 29511 W e.fire.dispatc: Skipping duplicate class check due to unsupported classloader
08-26 00:01:56.681 29511 29511 I Google Maps Android API: Google Play services client version: 12211000
08-26 00:01:56.685 29511 29511 I Google Maps Android API: Google Play services package version: 18719037
08-26 00:01:56.774 29511 29520 W System  : A resource failed to call close.
--------- beginning of crash
08-26 00:01:56.892 29511 29857 E AndroidRuntime: FATAL EXCEPTION: Thread-12
08-26 00:01:56.892 29511 29857 E AndroidRuntime: Process: com.timbond.seattle.fire.dispatch, PID: 29511
08-26 00:01:56.892 29511 29857 E AndroidRuntime: java.lang.NoClassDefFoundError: Failed resolution of: Lorg/apache/http/ProtocolVersion;
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at bx.b(:com.google.android.gms.dynamite_mapsdynamite@18719081@18.7.19 (100400-262610125):3)
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at by.a(:com.google.android.gms.dynamite_mapsdynamite@18719081@18.7.19 (100400-262610125):3)
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at ca.a(:com.google.android.gms.dynamite_mapsdynamite@18719081@18.7.19 (100400-262610125):14)
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at com.google.maps.api.android.lib6.drd.ak.a(:com.google.android.gms.dynamite_mapsdynamite@18719081@18.7.19 (100400-262610125):6)
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at bc.a(:com.google.android.gms.dynamite_mapsdynamite@18719081@18.7.19 (100400-262610125):22)
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at bc.run(:com.google.android.gms.dynamite_mapsdynamite@18719081@18.7.19 (100400-262610125):8)
08-26 00:01:56.892 29511 29857 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "org.apache.http.ProtocolVersion" on path: DexPathList[[zip file "/data/user_de/0/com.google.android.gms/app_chimera/m/00000068/MapsDynamite.apk"],nativeLibraryDirectories=[/data/user_de/0/com.google.android.gms/app_chimera/m/00000068/MapsDynamite.apk!/lib/arm64-v8a, /system/lib64]]
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at ad.loadClass(:com.google.android.gms.dynamite_dynamiteloader@18719081@18.7.19 (100400-262610125):4)
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
08-26 00:01:56.892 29511 29857 E AndroidRuntime:        ... 6 more
--------- beginning of system
08-26 00:01:56.901 29511 29857 I Process : Sending signal. PID: 29511 SIG: 9
```
### Environment

- Tabris.js version: 2.8.0
- tabris-plugin-maps@^5.0.0
- Device: Google Pixel 2 XL (taimen), Samsung Galaxy S10+ (beyond2q), Google Pixel 3 XL (crosshatch)
- OS: All Android 9

**Comment by mpost** [comment]
That is interesting. The apache http client has been [removed from Android in Android 6](https://developer.android.com/about/versions/marshmallow/android-6.0-changes#behavior-apache-http-client). I wonder how the play service all of a sudden would rely on them? We have tested on Android versions higher than 6. It might be worthwhile to update the maps library dependency.

**Comment by cookieguru** [comment]
@mpost Upgrade to what?  `5.0.0` is the latest version of the plugin.  I tried a production build with 2.8.1 and see the same issue.

**Comment by mpost** [comment]
sorry i meant to to upgrade the native maps library dependency.

https://developers.google.com/android/guides/releases
https://developers.google.com/maps/documentation/android-sdk/releases

**Comment by cookieguru** [comment]
@mpost Good call:
> After updating to com.google.android.gms:play-services-maps:16.1.0 you can remove this line from your own AndroidManifest.xml

Specifically referencing `org.apache.http.legacy` [here](https://developers.google.com/maps/documentation/android-sdk/releases#february_6_2019).

I updated the Play Services version to `16.1.0` but are now getting [some Kotlin errors](https://pastebin.com/raw/xKiU6xcT) when building the app.

**Comment by mpost** [comment]
Thanks for the info. i will take a look at it.

**Comment by cookieguru** [comment]
Clarification:

When I tried bumping the Play Services version to 16.1.0, my fork was based off f573665834d1ca66f088c3f3d49bc95382f00e95.  The version bump was the only change I made; `org.apache.http.legacy` was already included.  The previously-linked app build error was the result.

Separately, I tried modifying [v5.0.0](https://github.com/eclipsesource/tabris-plugin-maps/releases/tag/v5.0.0) (Play Services 12.0.1) to include `org.apache.http.legacy`.  The app built fine but experienced the same runtime crash.

Finally, I tried v5.0.0 with the Play Services version bump to 16.1.0 but without any other changes.  It works!  So it looks like there is a problem with the plugin in HEAD
